### PR TITLE
cloning now requires organs

### PR DESCRIPTION
**Pull request by SomeguyManperson**
# General Documentation

### Intent of your Pull Request

cloning will now require a healthy organic heart liver and pair of lungs to begin per clone
The clone pod (thing that grows the clone for the uneducated) can have these organs inserted into it by hitting it with them, only one of each can be stored at a time and they will still decay while inside the pod meaning storage should not be inside the pod when it is not in active use
hitting the pod with a fresh organ will dump the current organ if it is decayed
hitting the pod with an empty hand will dump all decayed organs

### Why is this change good for the game?
have fun learning surgery "why not just clone" people
# Wiki Documentation

### Briefly describe your PR and the impacts of it, in layman's terms. 
This will be the basis of the Wiki entry for your PR, and more information / detail is better for Wiki editors to integrate.
cloning now requires the clone pod to have a healthy heart liver and lung set inserted into it, organs inside the clone pod will still decay however
### What should players be aware of when it comes to the changes your PR is implementing?
to insert an organ into the cloning pod hit it while not on harm intent
to remove any decayed organs from the cloning pod hit it with an empty hand or a replacement organ
ONLY organic organs work, synthetic organs aren't accepted
### What general grouping does this PR fall under? 
Medical rebalancing

# Changelog

:cl:  
tweak: the cloning pod will now have to have a heart liver and lungs added to it to clone someone, with the organs decaying over time like normal
/:cl:

### CODE REVIEW

**Comment by Marmio64** [comment]
having it require meat encourages more interaction with other people like having to talk to chemistry for synthmeat or the chef for some monkey meat/gibbed humans, so I believe that the meat one is a better PR.

**Comment by SomeguyManperson** [comment]
having it take organs requires you to do actual medbay stuff to use cloning

**Comment by Tipy1802** [comment]
Better than meat
Wish it was something more to do with genetics though

**Comment by alexkar598** [comment]
Might as well remove the cloner then? I dont see what's the point of it. When is someone ever going to use the cloner after this change 

When medbay is not busy, cloning will be a task that takes too much time so theres no point cloning over defib

When medbay is busy, cloning will be a task that takes too much time so letting bodies rot while you tend to another is more efficient than cloning the ones you cant handle.


The meat PR does its job better, it discourages cloning, adds interaction while still allowing an equipped medbay to deal with emergencies by stocking piling meat. This PR is just a soft removal by making it nearly useless, only useful for husked and detached brains.

**Comment by SomeguyManperson** [comment]
Purpose of this is to wave the existence of surgery and organ replacement in front of anyone who decides to use cloning as a first line for literally everything, as the existence of defibs and alternwte revival methods is unknown to a large portion of the playerbase
What is also unknown to a large portion of the playerbase is evidently that the organ smartfridge will freeze organs so they can be "stockpiled" if necessary, adding something for normal medical doctors to do when not doing anything else (they could already do it but it wasnt woven into the meta now it is), and the existence of organs and the necessity of having spares makes going from "this guy has failing organs" to "oh we can replace those" much less difficult to think
In terms if interdepartmental interaction instead of talking with chem and the cook you are talking with botany and genetics and once in a blue moon maybe xenobio
To summarize
Regular medical players: mostly uneffected, have more reason to do surgery on monkeymen while there is downtime
New medical players: pushed to learn what surgery is and how to do it, showing how to recover a body without actually cloning it while in the process of preparing cloning for bodies. Also likely to be taught how an organ smartfridge works to combat organ decay
The cloner fail states for this are fairly to the point as to what they are caused by and I can probably make the cloning pod itself more intuitive in explaining itself

**Comment by SomeguyManperson** [comment]
missing/damaged organs are now displayed on examine
image
italics have been fixed since don't mind them

**Comment by Moggram** [comment]
And what if they don't requires organs like Plasmaman, Ethereal, Skeleton (I think they can be cloned), Slimepeople, and Preternis who have robo organs?

This PR is dum.

**Comment by SomeguyManperson** [comment]
ethereals and plasmamen have normal organs and the point isn't to make sense on a species by species basis it's to make cloning harder to do without actually putting in effort

**Comment by Darkstick** [comment]
Do cybernetic organs fit in the cloner?

**Comment by Moggram** [comment]
> 
> 
> Do cybernetic organs fit in the cloner?

It only takes organic organs.
